### ISSUE TITLE
No content with the version 0.1.3

### ISSUE THREAD

**Issue by EnricowithR** [hafen/trelliscopejs#17]
the package trelliscopejs is working correctly on the version 0.1.2 but with the version 0.1.3 (necessary to use the plotly functions - http://ryanhafen.com/blog/trelliscopejs-plotly) it does not work.

The version 0.1.3 shows the trelliscopejs frame without any content.
Maybe is there a bug on the version 0.1.3 that was not there on the version 0.1.2?

**Comment by hafen** [comment]
I've run over all the examples I have and everything works fine for me except for one case using `self_contained = TRUE` with htmlwidgets as panels, which should have been an issue prior to 0.1.3 as well. I see this as more of an edge case but I'll get it working soon. If you have other examples outside of this case that don't work, please provide a reproducible example.

**Comment by EnricowithR** [comment]
Yes, I am using self_contained = TRUE because I am using it inside Rmarkdown and Flexdashboard.

Below there is the code that is working with the version 0.1.2 (the same code with the version 0.1.3 does not work)
ggplot(comparison_DF,aes(x=variable,y=value)) + geom_bar(stat="identity", position = "dodge", fill = "#3F8411") +xlab("")+ylab(" ")+geom_text(aes(label=value),hjust=-.1)+ coord_flip() + theme(axis.title = element_text(size = 90),axis.text.y=element_text(colour="black", size = 15)) +facet_trelliscope(~ Course_name,name = "Attendance rate", self_contained = TRUE,  width = 1200, nrow = 2, ncol = 2, path = "f")

browseURL(file.path("f", "index.html"))

**Comment by hafen** [comment]
Thanks for the example. I'll take a look and hopefully have a fix soon.

**Comment by hafen** [comment]
This should be fixed in 0.1.4, which is now available in master (just use `install_github("hafen/trelliscopejs")` to get the latest.

Note that if you set the path appropriately, you're not required to use `self_contained=TRUE` in R Markdown documents.

**Comment by EnricowithR** [comment]
It is a bit better because it shows the titles of the graphs but the
content of the graphs it is not displayed yet.
I send you in attachment a screenshot of the result.

Plus for any versions the number of the graphs at the top right corner is
displayed with a strange character that it should not be there:

1â€“4 of 4

2016-12-31 2:31 GMT+01:00 hafen <notifications@github.com>:

**Comment by hafen** [comment]
The strange characters should be gone now if you install the latest.

Can you give me more details about your setup? What OS are you on? What web browser are you using? How are you viewing the output? Also a completely reproducible example that exhibits the issue would be helpful (for example, in your code provided above, I don't have "comparison_DF", so I can't run the example). If you can reproduce the issue with a built-in dataset in R, that would be helpful.

Also, the screenshot did not come through.

Thanks!

**Comment by EnricowithR** [comment]
With the latest version is working smoothly!
Thank you!

2017-01-02 21:07 GMT+01:00 hafen <notifications@github.com>:
